code/modules/events/wormholes.dm

```diff
@@ -17,7 +17,7 @@
 		if((T.z in config.station_levels))
 			pick_turfs += T
 
```

Comment:
Very likely, thought that would involve generating conflicts for your PR (also I don't feel like doing that in this PR =p)